zhaquirks/xiaomi/aqara/light_acn.py

```diff
@@ -84,3 +97,104 @@ class LumiLightAcn003(XiaomiCustomDevice):
             }
         }
     }
+
+
+class LumiLightAcn014(XiaomiCustomDevice):
+    """Quirk for Aqara LED Bulb T1 also known as Xiaomi ZNLDP13LM.
+
+    It identifies itself as lumi.light.acn014, however someone reported there are devices with the same model number
+    but identification lumi.light.cwac02.
+    Provides dimmable light control with color temperature setting.
+    This quirk adds support for power on behavior by adding power_outage_memory attribute.
+    """
+
+    signature = {
+        MODELS_INFO: [
+            (LUMI, "lumi.light.acn014"),
+        ],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.COLOR_TEMPERATURE_LIGHT,
+                INPUT_CLUSTERS: [
+                    BasicCluster.cluster_id,  # 0x0000
+                    DeviceTemperature.cluster_id,  # 0x0002
+                    Identify.cluster_id,  # 0x0003
+                    Groups.cluster_id,  # 0x0004
+                    Scenes.cluster_id,  # 0x0005
+                    OnOff.cluster_id,  # 0x0006
+                    LevelControl.cluster_id,  # 0x0008
+                    Alarms.cluster_id,  # 0x0009
+                    AnalogInput.cluster_id,  # 0x000C
+                    BinaryInput.cluster_id,  # 0x000F
+                    MultistateInput.cluster_id,  # 0x0012
+                    Color.cluster_id,  # 0x0300
+                    Metering.cluster_id,  # 0x0702
+                    ElectricalMeasurement.cluster_id,  # 0x0B04
+                    OppleClusterLight.cluster_id,  # 0xFCC0 - manufacturer specific
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,  # 0x000A
+                    Ota.cluster_id,  # 0x0019
+                ],
+            },
+            21: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.COLOR_TEMPERATURE_LIGHT,
+                INPUT_CLUSTERS: [
+                    AnalogInput.cluster_id,  # 0x000C
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
+            31: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.COLOR_TEMPERATURE_LIGHT,
+                INPUT_CLUSTERS: [
+                    AnalogInput.cluster_id,  # 0x000C
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
```

Comment:
As a note, these clusters are removed, since they're not in the replacement.
IIRC, Xiaomi normally uses those those endpoint numbers for power measurement plugs. Since this bulb isn't one though, these can likely be removed as well (which is already done).